src/renderer/shared/ui-kit/Combobox/Combobox.tsx

```diff
@@ -41,14 +42,24 @@ type InputProps = Pick<
 >;
 
 type ControlledPopoverProps = {
-  value: string;
+  selectedValue: string;
```

Comment:
If we want to store compound selected value it will also be visible in `Combobox Input`
To avoid this we separate input's value and selected one
```javascript
inputValue={signatoryAddress}
selectedValue={`${signatoryAddress}_${selectedWalletId}`}
```